doc/source/timeseries.rst

```diff
@@ -2145,94 +2147,86 @@ By default, pandas objects are time zone unaware:
    rng = pd.date_range('3/6/2012 00:00', periods=15, freq='D')
    rng.tz is None
 
-To supply the time zone, you can use the ``tz`` keyword to ``date_range`` and
-other functions. Dateutil time zone strings are distinguished from ``pytz``
-time zones by starting with ``dateutil/``.
+To localize these dates to a time zone, you can use the ``tz_localize`` method
```

Comment:
maybe have some definitions here, you jump right into localization w/o explaining what it is